docs/configuring-metabase/caching.md

```diff
@@ -24,7 +26,7 @@ These policies determines how long cached results will be stored.
 
 {% include plans-blockquote.html feature="Duration caching policy" %}
 
-Keep the cache for a number of hours. When someone runs a query, Metabase will first check if it's cached the results. If not, it runs the query and caches the results for as long as you set the duration.
+Keep the cache for X number of hours. When someone runs a query, Metabase will first check if it's cached the results. If not, it runs the query and caches the results for as long as you set the duration.
```

Comment:
Maybe:

`Keep the cache for a certain number of hours.`

Maybe this is just me but I trip a little over "it's cached" because I first think that's short for "it is"

Reply:
Perhaps "caches the results for the specified duration"?